R/data.table.R

```diff
@@ -62,7 +62,7 @@ data.table = function(..., keep.rownames=FALSE, check.names=FALSE, key=NULL, str
   if (!is.null(key)) {
     if (!is.character(key)) stop("key argument of data.table() must be character")
     if (length(key)==1L) {
-      key = strsplit(key,split=",")[[1L]]
+      key = trimws(strsplit(key,split=",")[[1L]])
```

Comment:
why not split on `,\\s*`?

please change to `fixed=TRUE` if we are going to keep `split=','`

Reply:
because that would not work for `key = " A,B "`, despite it's trivial.

Sure, added.

Reply:
`strsplit(trimws(key), split = ',\\s*')` works though 🙃 

i'm fine with either this or the `fixed=TRUE` approach, anyway